neo/Consensus/ConsensusService.cs

```diff
@@ -514,6 +517,7 @@ public static Props Props(IActorRef localNode, IActorRef taskManager, Store stor
 
         private void RequestChangeView()
         {
+            if (context.WatchOnly()) return;
```

Comment:
I had missed that. Nice catch, @erikzhang.
I think it may happen if transactions has problem at AddTransaction.